### PR TITLE
Nearest User Node

### PR DESCRIPTION
**Pull request by LeCloutPanda**
Nearest user node

For issues [57](https://github.com/Xlinka/NeosPlus/issues/57) and [39](https://github.com/Xlinka/NeosPlus/issues/39)

### CODE REVIEW

**Comment by Xlinka** [comment]
Use More Appropriate Names when doing pull requests and what the node is in the title and not use "Deez"

**Comment by LeCloutPanda** [comment]
Sorry :)

File: NEOSPlus.csproj [hunk 1]
```diff
@@ -109,6 +110,8 @@
     <Compile Include="Logix\NodeExtensions.cs" />
     <Compile Include="Logix\Operators\ZeroOneI.cs" />
     <Compile Include="Logix\Math\SortNode.cs" />
+    <Compile Include="Logix\Slots\FindAllChildrenWithTag.cs" />
```
Comment (Frozenreflex) [inline NEOSPlus.csproj:113]: Remove these two

**Review by Frozenreflex — CHANGES_REQUESTED** [review]

File: NEOSPlus.csproj (hunk 1 above)
Comment (LeCloutPanda) [reply NEOSPlus.csproj]: Should be gone now.

**Review by Frozenreflex — APPROVED** [review]

**Pull request merged**